docs/use_cases/apps.md

```diff
@@ -7,8 +7,8 @@ LlamaIndex modules provide plug and play data loaders, data structures, and quer
 Chatbots are an incredibly popular use case for LLM's. LlamaIndex gives you the tools to build Knowledge-augmented chatbots and agents.
 
 Relevant Resources:
-- [Building a Chatbot](/guides/tutorials/building_a_chatbot.md)
```

Comment:
Is this broken? Looks fine to me on https://gpt-index.readthedocs.io/en/latest/use_cases/apps.html